ts/WoltLabSuite/Core/Ui/ItemList/LineBreakSeparatedText.ts

```diff
@@ -23,10 +23,20 @@ export class UiItemListLineBreakSeparatedText {
   protected readonly itemList: HTMLUListElement;
   protected readonly items = new Set<string>();
   protected readonly options: LineBreakSeparatedTextOptions;
+  protected readonly submitField?: HTMLInputElement = undefined;
 
-  constructor(itemList: HTMLUListElement, options: LineBreakSeparatedTextOptions) {
+  constructor(itemList: HTMLUListElement, options?: LineBreakSeparatedTextOptions) {
     this.itemList = itemList;
-    this.options = options;
+    this.options = options || {};
```

Comment:
```suggestion
    this.options = options;
```
See the default value of the `options` parameter in the suggested change for line 28 above.